### PR TITLE
LED-143 Collapse: new implementation

### PR DESCRIPTION
**Pull request by tom-in-the-shell**

### CODE REVIEW

File: docs/src/app/layout-components/collapse/MainDemo/BasicDemo.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { Live } from '@/components/live';
+
+export const BasicDemo = () => (
+  <Live scope={{ L }}>
+    {`
+() => {
+  return (
+    <>
+      <L.Collapse
+        summary={'Summary'}
```
Comment (777PolarFox777) [inline docs/src/app/layout-components/collapse/MainDemo/BasicDemo.tsx]:
```suggestion
        summary="Summary"
```

File: docs/src/app/layout-components/collapse/MainDemo/ControlledDemo.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        onClick={() => setIsOpen(!isOpen)}
+        _mb-4
+      >
+        Toogle Collapse
+      </L.Button>
+
+      <L.Collapse
+        isOpen={isOpen}
+        onToggle={(ev) => setIsOpen(ev.isOpen)}
+        summary={'Summary'}
```
Comment (777PolarFox777) [inline docs/src/app/layout-components/collapse/MainDemo/ControlledDemo.tsx]:
```suggestion
        summary="Summary"
```

File: leda/components/Collapse/index.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  const detailsClassName = classnames(
+    theme.wrapper,
+    className,
+  );
+
+  return (
+    <details
+      className={detailsClassName}
+      // @ts-expect-error name attribute in not present in the details types
+      name={accordionId}
```
Comment (777PolarFox777) [inline leda/components/Collapse/index.tsx]: Won't DOM print working in that case?

**Review by 777PolarFox777 — APPROVED** [review]

File: leda/components/Collapse/index.tsx (hunk 3 above)
Comment (tom-in-the-shell) [reply leda/components/Collapse/index.tsx]: I forgot to remove it, removed

**Pull request merged**